color_mapn_test.cpp

```diff
@@ -34,6 +34,7 @@ int main() {
         "25 Violet Slate"
     };
     assert(result == colorCombinations);
-    std::cout << "All is well\n";
+    printColormap();

```

Comment:
Why are there two functions, one printColormap and another createColormap?
One is tested automatically, other is tested manually. You can re-use createColormap inside printColormap instead?